utils/strings/validations.ts

```diff
@@ -1,3 +1,14 @@
 export function isEmpty(value: string) {
   return !value.trim();
 }
+
+// Note: This denotes a url as valid even without a protocol or domain.
+export function isValidUrl(value: string) {
+  const patternWithProtocol = new RegExp(
+    '^(http[s]?:\\/\\/(www\\.)?|ftp:\\/\\/(www\\.)?|www\\.){1}([0-9A-Za-z-\\.@:%_+~#=]+)+((\\.[a-zA-Z]{2,3})+)(/(.)*)?(\\?(.)*)?',
+  );
+
+  const patternWithoutProtocol = new RegExp('^([0-9A-Za-z-\\.@:%_+~#=]+)+((\\.[a-zA-Z]{2,3})+)(/(.)*)?(\\?(.)*)?');
```

Comment:
If I'm reading this correctly, the TLD is specified as only being 2-3 characters right?

But I think `.bank`, `.google` and other TLDs are valid now

Reply:
When I tested it works even with longer TLD? Actually I'm also not so sure about regex :   )